www/_posts/2017-03-06-plugins-release.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today:
+
+* cordova-plugin-battery-status@1.2.3
+* cordova-plugin-camera@2.4.0
+* cordova-plugin-console@1.0.6
+* cordova-plugin-contacts@2.3.0
+* cordova-plugin-device-motion@1.2.4
+* cordova-plugin-device-orientation@1.0.6
+* cordova-plugin-device@1.1.5
+* cordova-plugin-dialogs@1.3.2
+* cordova-plugin-file-transfer@1.6.2
+* cordova-plugin-file@4.3.2
+* cordova-plugin-geolocation@2.4.2
+* cordova-plugin-globalization@1.0.6
+* cordova-plugin-inappbrowser@1.7.0
+* cordova-plugin-media@3.0.0
+* cordova-plugin-media-capture@1.4.2
+* cordova-plugin-network-information@1.3.2
+* cordova-plugin-splashscreen@4.0.2
+* cordova-plugin-statusbar@2.2.2
+* cordova-plugin-vibration@2.1.4
+* cordova-plugin-whitelist@1.3.2
+* cordova-plugin-wkwebview-engine@1.1.2
+* cordova-plugin-test-framework@1.1.5
+
+Release Highlights:
+
+* **CAMERA:** [CB-12005](https://issues.apache.org/jira/browse/CB-12005) Changing the `getOrientation` method to return the defined enumerated `EXIF` instead of orientation in degrees for Consistency
+* **In App Browser:** [CB-9148](https://issues.apache.org/jira/browse/CB-9148) **Android:** Add Support for `input[type=file]` File Chooser
```

Comment:
Hmm, in the [inappbrowser plugin's release notes, we refer to it as `InAppBrowser`](https://github.com/apache/cordova-plugin-inappbrowser/blob/master/RELEASENOTES.md#170-feb-28-2017). Which should be used?